core/views.py

```diff
@@ -145,11 +139,7 @@ def year_2016_2017(request):
         request,
         "core/2016-2017.html",
         {
-            "events_2015": Event.objects.public().filter(date__lt="2016-01-01").order_by("date"),
```

Comment:
This separate query wasn't used in the template at all. Must have been some kind of a leftover.